testclient/mainwindow.cpp

```diff
@@ -263,6 +264,15 @@ void MainWindow::printPBClicked()
     }
 }
 
+void MainWindow::stopPBClicked()
+{
+    if (core->state() == BUSY) {
```

Comment:
It's better to send only the stop command to the printer. Don't have problem if you send more than one.